Index: lib/capistrano/cli.rb
===================================================================
--- lib/capistrano/cli.rb	(revision 5636)
+++ lib/capistrano/cli.rb	(working copy)
@@ -289,6 +289,12 @@
       end
 
       def look_for_default_recipe_file!
+        if File.exist?(system_defaults = '/etc/capistrano.conf') 
+          @options[:recipes] << system_defaults
+        end
+        if File.exist?(user_defaults = File.join(ENV['HOME'],'.caprc')) 
+          @options[:recipes] << user_defaults
+        end
         DEFAULT_RECIPES.each do |file|
           if File.exist?(file)
             @options[:recipes] << file
